lib/all_unique.js

```diff
@@ -1,8 +1,18 @@
 module.exports = {
   all_unique: function(str){
-  
+  var separatedString = str.slice(0);
+  for (currentChar = 0; currentChar < separatedString.length; currentChar++){
+    var equalCharCount = 0;
+    for (nextChar = 0; nextChar < separatedString.length; nextChar++){
+    if (separatedString[currentChar] === separatedString[nextChar])
+      equalCharCount += 1;
+  }
+  if (equalCharCount === 0)
+    return true;
+  else
+    return false;
+    }
   }
-}
 
 /* Algorithmic Thought Process:
```

Comment:
[Multi line comments](http://www.w3schools.com/js/js_comments.asp) are written using /* and */. At least that's what I've read. I'll try improving indentation and commit again.